### ISSUE TITLE
AAF Writer missing from setup.py

### ISSUE THREAD

**Issue by alatdneg** [PixarAnimationStudios/OpenTimelineIO#452]
File opentimelineio_contrib/adapters/advanced_authoring_format.py", line 46, in <module>
    from opentimelineio_contrib.adapters.aaf_adapter import aaf_writer  # noqa: E402
ImportError: No module named aaf_adapter

**Comment by alatdneg** [comment]
```
    packages=[
        'opentimelineio',
        'opentimelineio.adapters',
        'opentimelineio.algorithms',
        'opentimelineio.core',
        'opentimelineio.schema',
        'opentimelineio.schemadef',
        'opentimelineio.plugins',
        'opentimelineio.console',
        'opentimelineio_contrib',
        'opentimelineio_contrib.adapters',
        'opentimelineio_contrib.adapters.aaf_adapter', MISSING..
        'opentimelineview',
    ],

```

**Comment by ssteinbach** [comment]
Good catch!  Thanks @alatdneg.

**Comment by ssteinbach** [comment]
Should have been addressed by #453.  Just to double check -- what method were you using to install OTIO that you hit this?  `python setup.py install`?  `pip install OpenTimelineIO` seemed to still work when I was testing it.
